cpp/src/io/parquet/experimental/hybrid_scan_impl.cpp

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "hybrid_scan_impl.hpp"
+
+#include "cudf/io/text/byte_range_info.hpp"
+#include "hybrid_scan_helpers.hpp"
+
+#include <cudf/detail/stream_compaction.hpp>
+#include <cudf/detail/transform.hpp>
+#include <cudf/detail/utilities/stream_pool.hpp>
+#include <cudf/io/parquet_schema.hpp>
+#include <cudf/strings/detail/utilities.hpp>
+#include <cudf/utilities/error.hpp>
+#include <cudf/utilities/memory_resource.hpp>
+
+#include <thrust/iterator/counting_iterator.h>
+
+#include <bitset>
+#include <iterator>
+#include <limits>
+#include <numeric>
+
+namespace cudf::io::parquet::experimental::detail {
+
+using ColumnChunkDesc       = parquet::detail::ColumnChunkDesc;
+using decode_kernel_mask    = parquet::detail::decode_kernel_mask;
+using PageInfo              = parquet::detail::PageInfo;
+using PageNestingDecodeInfo = parquet::detail::PageNestingDecodeInfo;
+using byte_range_info       = text::byte_range_info;
```

Comment:
As above, you can simplify these using declarations.
```suggestion
using parquet::detail::ColumnChunkDesc;
using parquet::detail::decode_kernel_mask;
using parquet::detail::PageInfo;
using parquet::detail::PageNestingDecodeInfo;
using text::byte_range_info;
```